airsonic-main/src/main/java/org/airsonic/player/service/PlayerService.java

```diff
@@ -88,11 +90,16 @@ public synchronized Player getPlayer(HttpServletRequest request, HttpServletResp
                                          boolean remoteControlEnabled, boolean isStreamRequest) {
 
         // Find by 'player' request parameter.
-        Player player = getPlayerById(request.getParameter("player"));
+        Player player;
+        try {
+            player = getPlayerById(ServletRequestUtils.getIntParameter(request, "player"));
+        } catch (ServletRequestBindingException e) {
+            player = null;
```

Comment:
This makes me a bit uneasy of silently throwing away this error here. Seems like the we should fix the exception getting thrown.

Reply:
Yes, I wasn't sure about that, I'll add 'throws Exception' after 'getPlayer' and remove the inline check if that works.